anaconda.spec.in

```diff
@@ -317,18 +324,23 @@ update-desktop-database &> /dev/null || :
 %config %{_sysconfdir}/%{name}/*
 %dir %{_sysconfdir}/%{name}/conf.d
 %config %{_sysconfdir}/%{name}/conf.d/*
-%ifarch %livearches
+
+%files live
 %{_bindir}/liveinst
 %{_sbindir}/liveinst
 %config(noreplace) %{_sysconfdir}/pam.d/*
 %config(noreplace) %{_sysconfdir}/security/console.apps/*
 %{_libexecdir}/liveinst-setup.sh
 %{_datadir}/applications/*.desktop
+%{_datadir}/anaconda/gnome
 %{_sysconfdir}/xdg/autostart/*.desktop
-%endif
 
 %files gui
 %{python3_sitearch}/pyanaconda/ui/gui/*
+%{_datadir}/anaconda/pixmaps
+%{_datadir}/anaconda/ui
+%{_datadir}/anaconda/window-manager
+%{_datadir}/anaconda/anaconda-gtk.css
```

Comment:
Have you tried to run anaconda without anaconda-gui?

Reply:
I think I should be able to try that on my Initial Setup VM & possibly we could try generating a text only boot.iso.